src/core/vr-scene.js

```diff
@@ -59,7 +60,26 @@
           function attachEventListener(node) {
             node.addEventListener('loaded', elementLoaded);
           }
-        }
+        },
+
+        attachFullscreenListeners: {
+          value: function() {
+            // handle fullscreen changes
+            document.addEventListener('fullscreenchange', this.fullscreenChange.bind(this), false);
```

Comment:
btw `useCapture` defaults to `false`, so if you want, you can remove the last arg